testprojects/tests/java/org/pantsbuild/testproject/resourcejar/ResourceJar.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package org.pantsbuild.testproject.resourcejar;
+
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Demonstrate that instead of a directory of resources on the classpath we put a zip of the
+ * resources on the classpath.
+ */
+public class ResourceJar {
+
+  @Test
+  public void testResourceJar() throws Exception {
+    assertEquals(
+      "1234567890",
```

Comment:
thought about that. the thing is because use-jar is by default disabled in OSS pants, if I assert jar here, it would be broken unless the tests is run with:

./pants tests testprojects/tests/java/org/pantsbuild/...../resourcesjar   --resources-prepare-use-jar

this testproject addition is mostly for manual testing.

like i said, I haven't added any unittests or IT tests for the changes yet. So this pull request isn't complete.